### PR TITLE
https://issues.redhat.com/browse/ACM-7136 Overview beta switch

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-7136

### CODE REVIEW

**Review by zlayne — APPROVED** [review]

File: console/console.adoc [hunk 1]
```diff
@@ -31,6 +31,8 @@ From the _Overview_ page, you can view the following information:
 * Pod status
 * Cluster add-ons
 
+You can also use the _Beta view_ switch from the _Overview_ page header to filter the page data by using cluster labels, and display metrics. If observability is enabled, alert and failing operator metrics from across your fleet are displayed.
+
```
Comment (dockerymick) [inline console/console.adoc:35]: Note to self: Wondering if it is best to also have a link to the Enable observability topic.

**Review by oafischer — APPROVED** [review]
Looks great!

File: release_notes/whats_new.adoc [hunk 2]
```diff
@@ -52,6 +52,8 @@ For other Application topics, see link:../applications/app_management_overview.a
 //2.9:6709
 * You can use the `InformOnly` parameter to check the status of Kubernetes objects without enforcing a configuration policy, even if the parent policy `remediationAction` is set to `enforce`. See link:../governance/config_policy_ctrl.adoc#kubernetes-configuration-policy-controller[Kubernetes configuration policy controller] to learn more.
```
Comment (dockerymick) [inline release_notes/whats_new.adoc:56]:
```suggestion

//2.9:6549
```

File: release_notes/whats_new.adoc [hunk 3]
```diff
@@ -37,7 +37,7 @@ View release notes, as well as tasks and support information at link:../clusters
 
 *Technology Preview:* Pull model is in a Technology Preview status, with limited support this release.
```
Comment (dockerymick) [inline release_notes/whats_new.adoc:39]:
```suggestion

//2.9:5868
```

File: release_notes/whats_new.adoc [hunk 4]
```diff
@@ -37,7 +37,7 @@ View release notes, as well as tasks and support information at link:../clusters
...
+* You can use the Argo CD pull model to deploy resources from your hub cluster to each managed cluster with the same `ApplicationSet` custom resource definition that is used for the push model. Pull model implementation applies {ocp} registration, placement, and `manifestWork` APIs so that the hub cluster can use the secure communication channel between the hub cluster and the managed cluster to deploy resources. See link:../applications/gitops_push_pull.adoc#argo-pull-push[Deploying Argo CD with the push and pull model] for more information.
 
 * You can now go to the topology page to view the `ApplicationSet` applications that were created by using the Argo CD pull model.
```
Comment (dockerymick) [inline release_notes/whats_new.adoc:44]:
```suggestion
//2.9:6830
* You can now go to the topology page to view the `ApplicationSet` applications that were created by using the Argo CD pull model. 
```

**Pull request merged**